### PR TITLE
Reduce the number of Docker repositories created

### PR DESCRIPTION
**Pull request by ehuelsmann**

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ jobs:
         image_tag: ${{ matrix.browser }}
         push_git_tag: ${{ github.event_name == 'push' || github.event.pull_request.merged }}
         context: ${{ matrix.browser }}
+        stages_image_name: "${{ github.repository_owner }}dev/ledgersmb_circleci-browsers-${{ matrix.browser }}-stages"
```
Comment (ylavoie) [inline .github/workflows/docker-image.yml:39]: That will force developers to get a ${{ github.repository_owner }}dev docker repository for them to test. Shouldn't we adjust for ledgersmbdev or ${{ github.repository_owner }} to simplify?

**Review by ylavoie — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (ehuelsmann) [reply .github/workflows/docker-image.yml:39]: Why is this line (although being added) any different than on line 36 where the same is indicated? Or on line 16?

Comment (ylavoie) [reply .github/workflows/docker-image.yml:39]: It's not, all 3 mandates developers to set up a dev repository on docker hub.

Comment (ehuelsmann) [reply .github/workflows/docker-image.yml:39]: Ok. But: if it requires the developer to either create a namespace on docker hub by the name of `$developer_name` or `${developer_name}dev`, what's the difference? I mean: both require a Docker Hub repository?

Comment (ehuelsmann) [reply .github/workflows/docker-image.yml:39]: They need a Docker Hub repository in their own name too. All in all not a practical setup. But I think it's the best we can do for now.

**Comment by ylavoie** [comment]
Integrated in #26

**Pull request closed without merging**